crowbar_framework/index/index.html

```diff
@@ -114,7 +116,10 @@
     </div>
 
     <script>
-      document.getElementById("dashboard").href="http://" + window.location.host + ":3000";
+      links_to_dashboard = document.getElementsByClassName("dashboard");
+      for (i = 0; i < links_to_dashboard.length; i++) {
+        links_to_dashboard[i].href="http://" + window.location.host + ":3000";
```

Comment:
Purely matter of style - the `i`th element can be assigned into a special variable. This is useful when the variable is accessed frequently inside the loop and also makes the code a tiny bit more readable:

```
var link = links_to_dashboard[i];
link.href = "...";
```